modules/installation-network-user-infra.adoc

```diff
@@ -111,3 +111,12 @@ A working configuration for the Ingress router is required for an
 {product-title} cluster. You must configure the Ingress router after the control
 plane initializes.
 ====
+
+[discrete]
+== Ethernet adaptor hardware address requirements
```

Comment:
@jboxman, this module's used in the bare metal assembly, too. You need to conditionalize this for the vSphere assembly or make it generic.